website/content/docs/enterprise/license/faqs.mdx

```diff
@@ -0,0 +1,102 @@
+---
+layout: docs
+page_title: Frequently Asked Questions (FAQs)
+description: An overview of license.
+---
+
+# Frequently Asked Questions (FAQs)
+
+This FAQs section is for the license changes introduced in Vault Enterprise 1.8. We plan to deprecate the old license system (vault write sys/license) where autoload will be the only way to manage your license.
+
+## Q: When will these licensing changes be released for Vault?
+
+The licensing and EULA changes will be available with Vault 1.8-RC1 on June 16, 2021. Vault 1.8 will be GA on July 28, 2021.
+
+## Q: Will these license changes impact HCP Vault?
+
+No, these changes will not impact HCP Vault.
+
+## Q: Do these license changes impact all Vault customers/licenses?
+
+| Customer/licenses                                                                  | Impacted? |
+| ---------------------------------------------------------------------------------- | --------- |
+| ENT binaries (Evaluation or non-Evaluation downloaded from releases.hashicorp.com) | Yes       |
+| Open-Source (OSS)                                                                  | No        |
+| Baked-in license binaries (pro/premium)                                            | No        |
```

Comment:
@aartiiyengar Correct me if I'm wrong, but based on the [Vault Enterprise Packaging](https://www.datocms-assets.com/2885/1621618395-vault-enterprise-packaging-fy22.pdf), the available packages are **Standard**, **Plus**, **Premium** and **ADP**.  

```suggestion
| Baked-in license binaries (plus/premium)                                            | No        |
```